squidb-tests/src/com/yahoo/squidb/sql/InsertTest.java

```diff
@@ -273,7 +273,7 @@ public void run() {
                 // insert into testModels (firstName, lastName) values ("Jack", "Sparrow"), ("James", "Bond", 007),
                 // ("Bugs", "Bunny");
                 Object[] values1 = new Object[]{"Jack", "Sparrow"};
-                Object[] values2 = new Object[]{"James", "Bond", Integer.valueOf(007)};
```

Comment:
Haha sorry. I was cleaning up the "unnecessary boxing" warnings and didn't notice the actual context here.